frontend/src/js/controllers/challengeCreate.js

```diff
@@ -70,7 +70,7 @@
                             utilities.hideLoader();
                             var error = response.data;
                             angular.element(".file-path").val(null);
-                            $rootScope.notify("error", error.error);
+                            $rootScope.notify("Error in creating challenge. Please check the yaml configuration!", error.error);
```

Comment:
Hi @calenrobinette , the first error shows that the notification type is of `error` & second one gives the message which is correct that is `error.error`.
We have to change it at the [backend](https://github.com/Cloud-CV/EvalAI/blob/master/apps/challenges/views.py#L548) after caching the error [here](https://github.com/Cloud-CV/EvalAI/blob/master/apps/challenges/views.py#L546)